Added service annoucement paragrapth (#3523)
SoftwareAG/c8y-docs: content/service-terms/service-level-bundle/platform-sla.md

(c) **Events of force majeure**.

### Service announcements

{{< company-c8y >}} communicates service availability and events impacting the service through the following status pages:

* [status.cumulocity.com](http://status.cumulocity.com) for US and EMEA instances.
* [cumulocity-apj.statuspage.io](http://cumulocity-apj.statuspage.io) for APJ instances.
* a dedicated location for {{< product-c8y-iot >}} Dedicated instances.

Planned and unplanned downtimes are communicated via the appropriate {{< product-c8y-iot >}} status page. The communication will include the expected time of service restauration.

> > **Note:** The availability of the status page itself is not included in the calculation of service availability.

Reviewer: ```suggestion
 {{< c8y-admon-info >}}
The availability of the status page itself is not included in the calculation of service availability.
{{< /c8y-admon-info >}}  
```

(c) **Events of force majeure**.

### Service announcements

{{< company-c8y >}} communicates service availability and events impacting the service through the following status pages:

* [status.cumulocity.com](http://status.cumulocity.com) for US and EMEA instances.
* [cumulocity-apj.statuspage.io](http://cumulocity-apj.statuspage.io) for APJ instances.
* a dedicated location for {{< product-c8y-iot >}} Dedicated instances.

> Planned and unplanned downtimes are communicated via the appropriate {{< product-c8y-iot >}} status page. The communication will include the expected time of service restauration.

Reviewer: ```suggestion
Planned and unplanned downtimes are communicated via the appropriate {{< product-c8y-iot >}} status page. The communication includes the expected time of service restoration.
```

>  {{< c8y-admon-info >}}

Reviewer: ```suggestion

{{< c8y-admon-info >}}
```

(c) **Events of force majeure**.

### Service announcements

{{< company-c8y >}} communicates service availability and events impacting the service via status pages. The status pages showing the service availability results are at

* [status.cumulocity.com](http://status.cumulocity.com) for US and EMEA instances.
* [cumulocity-apj.statuspage.io](http://cumulocity-apj.statuspage.io) for APJ instances.
* a dedicated location for {{< product-c8y-iot >}} Dedicated instances.

Planned and unplanned downtimes for the services are communicated via the {{< product-c8y-iot >}} status page, which will also provide an expected time for the system’s return to availability. Please note that the availability of the status page itself is not included in the services availability calculations.

> Service impacting changes and planned downtime will be announced via the {{< product-c8y-iot >}} status page latest 14 days prior to the event. Updates with additional details may be published closer to the time of the event. 

Reviewer: ```suggestion
Service-impacting changes and planned downtimes are announced via the {{< product-c8y-iot >}} status page at least 14 days prior to the event. Updates with additional details may be published closer to the time of the event. 
```

> **Please note:** Emergency maintenances might be announced ad-hoc or with a shorter announcement period.   

Reviewer: ```suggestion
{{< c8y-admon-info >}}
Emergency maintenance might be announced ad-hoc or with a shorter announcement period. 
{{< /c8y-admon-info >}}  
```

(c) **Events of force majeure**.

### Service announcements

> {{< company-c8y >}} communicates service availability and events impacting the service via status pages. The status pages showing the service availability results are at

Reviewer 2: ```suggestion
{{< company-c8y >}} communicates service availability and events impacting the service through the following status pages:
```

* [status.cumulocity.com](http://status.cumulocity.com) for US and EMEA instances.
* [cumulocity-apj.statuspage.io](http://cumulocity-apj.statuspage.io) for APJ instances.
* a dedicated location for {{< product-c8y-iot >}} Dedicated instances.

> Planned and unplanned downtimes for the services are communicated via the {{< product-c8y-iot >}} status page, which will also provide an expected time for the system’s return to availability. Please note that the availability of the status page itself is not included in the services availability calculations.

Reviewer 2: ```suggestion
Planned and unplanned downtimes are communicated via the appropriate {{< product-c8y-iot >}} status page. The communication will include the expected time of service restauration.
> **Note:** The availability of the status page itself is not included in the calculation of service availability.
```

> Service-impacting changes and planned downtimes are announced via the {{< product-c8y-iot >}} status page at least 14 days prior to the event. Updates with additional details may be published closer to the time of the event. 

Reviewer: ```suggestion
Service-impacting changes and planned downtimes are announced via the {{< product-c8y-iot >}} status page at least 14 days prior to the event. Updates with additional details might be published closer to the time of the event. 
```

> Service-impacting changes and planned downtimes are announced via the {{< product-c8y-iot >}} status page at least 14 days prior to the event. Updates with additional details might be published closer to the time of the event. 

Reviewer 2: ```suggestion
Service-impacting changes and planned downtimes are announced on the {{< product-c8y-iot >}} status page at least **14 days prior to the event**. Additional updates with more detailed information may be provided as the event approaches. 
```

{{< c8y-admon-info >}}

> Emergency maintenance might be announced ad-hoc or with a shorter announcement period. 

Reviewer 2: ```suggestion
Emergency maintenance may be announced on short notice or without prior warning. 
```
